docs/API/metadata.md

```diff
@@ -104,14 +104,20 @@ received based on user profile preferences and server configuration; but it can
 be modified by the owner of the dataset, as long as the new timestamp remains
 within the maximum allowed server data retention period.
 
-### Dashboard namespace
+### Global namespace
 
 The server will never modify or directly interpret values in this namespace. An
 authenticated client representing the owner of a dataset can set any keys
 within this namespace to any valid JSON values (string, number, boolean, list,
 or nested objects) for retrieval later. All clients with read access to the
 dataset will see the same values.
 
+The recommended best practice is to assign a project sub-key that will be unique
+and avoid collisions between distinct clients. The Pbench Dashboard project,
```

Comment:
I don't think "assign" is the right word.  We don't have or envision any sort of registrar who would be responsible for giving out such assignments.  I suggest using "select" (or "use", or similar) in place of "assign", here and at line 140.